### PR TITLE
Add asssets/images folder, shorten image paths, and fix heading consistency

### PR DESCRIPTION
**Pull request by charleslavoy**

### CODE REVIEW

File: labs/apic-em-path-trace/2.md [hunk 1]
```diff
@@ -104,11 +104,11 @@ Status:  200
...
-### Session summary
+## Session summary
 
 * The `host` API works similarly to the `network-device` API. You can use the host API to find out how many hosts are in Inventory, how each host connects to the network, and much more.  <br>
 * In the next lab session, you apply what you learned in this session, putting it all together to work with a new API, the `/flow-analysis` API.
 
-### Next Step:
+## Next step
 
 * Proceed to Step 2: Use Path Trace.
```
Comment (annegentle) [inline labs/apic-em-path-trace/2.md]: Step 3 is Using Path Trace

File: labs/apic-em-path-trace/3.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
-## Step 3: Using Path Trace
+# Step 3: Using Path Trace
```
Comment (annegentle) [inline labs/apic-em-path-trace/3.md]: Using or Use?

File: labs/apic-em-path-trace/3.md [hunk 3]
```diff
@@ -639,8 +639,8 @@ You learned how to use the `POST /flow-analysis` API to trace a path between two
 
 Although this example is text-based for the sake of simplicity, what you learned in this session can be used in an application that shows path trace results graphically, like the following example. This screen is from the `Path Trace` application in the Cisco APIC-EM GUI, which uses these very same APIs to do its work.
 
-![Figure](/posts/files/apic-em-path-trace/apicem_routing_path.jpg)
+![Figure](assets/images/apicem_routing_path.jpg)
 
-#### <font color='red'>Note: Because you are using DB only sandbox, not all source/destination pairs are able to retrieve the trace path. </font>
+### <font color='red'>Note: Because you are using DB only sandbox, not all source/destination pairs are able to retrieve the trace path. </font>
```
Comment (annegentle) [inline labs/apic-em-path-trace/3.md]: using a database-only Sandbox, perhaps?

File: labs/apic-em-path-trace/2.md (hunk 1 above)
Comment (charleslavoy) [reply labs/apic-em-path-trace/2.md]: Fixed. 👍

File: labs/apic-em-path-trace/3.md (hunk 2 above)
Comment (charleslavoy) [reply labs/apic-em-path-trace/3.md]: Fixed. Missed that one. Good catch. 👓

File: labs/apic-em-path-trace/3.md (hunk 3 above)
Comment (charleslavoy) [reply labs/apic-em-path-trace/3.md]: Fixed. Thanks!

**Pull request merged**